Makefile

```diff
@@ -13,7 +13,43 @@ FRONTEND_DOCKER_IMAGE ?= frontend
 FLATTERNER_DOCKER_IMAGE ?= query-service
 QUERY_SERVICE_DOCKER_IMAGE ?= flattener-processor
 
-all: build-push-frontend build-push-query-service build-push-flattener
+all: frontend-service-checks backend-services-checks build-push-frontend build-push-query-service build-push-flattener
+
+frontend-service-checks:
+	@echo "------------------"
+	@echo "--> Checking unused imports of signoz's frontend [depcheck]"
```

Comment:
What is the motive for frontend checks? What do we want to achieve? 🤔

Reply:
In this PR, I have removed all the unused imports from the front end. So, this check will restrict developers from pushing unused dependencies to the master branch.

This is a  depcheck that will list all unused dependencies.